pinot/inference/gp/kernels.py

```diff
@@ -0,0 +1,47 @@
+#=============================================================================
+# IMPORTS
+# =============================================================================
+import torch
+import pinot
+import abc
+from pinot.inference.gp.kernel import Kernel
+
+# =============================================================================
+# MODULE CLASSES
+# =============================================================================
+class RBF(Kernel):
+    r""" A Gaussian Process Kernel that hosts parameters.
+
+
+    """
+    def __init__(self, l=1.0):
+        super(RBF, self).__init__()
+        self.l = l
+
+    def distance(self, x, x_):
+        return torch.norm(
+                x[:, None, :] - x_[None, :, :],
+                p=2,
+                dim=2)
+
+    def forward(self, x, x_=None, l=None):
```

Comment:
Some comments about what this function is doing, from a high level point of view would be appreciated. I'm not familiar with deep GP.